lib/jsdom/living/nodes/HTMLInputElement-impl.js

```diff
@@ -95,6 +95,11 @@ function valueAttributeMode(type) {
   return "value";
 }
 
+function getTypeFromAttribute(typeAttribute) {
+  const type = typeAttribute && typeAttribute.toLowerCase();
```

Comment:
This is probably a preexisting problem elsewhere in the file, but I believe this will give the wrong result for "ſearch" where the "ſ" is "LATIN SMALL LETTER LONG S", or "weeK" where the "K" is "KELVIN SIGN".

Other k or s-containing types are checkbox, reset, password, submit... I think that's all of them.

Reply:
Indeed, I got the code from the original `type` getter. I'll fix it.